rstcheck.py

```diff
@@ -688,6 +688,16 @@ def visit_literal_block(self, node):
 
         raise docutils.nodes.SkipNode
 
+    def visit_paragraph(self, node):
+        """Check syntax of reStructuredText."""
+        find = re.search(r'\[[^\]]*\]\([^\)]*\)', str(node))
```

Comment:
Let's require at least one character in the link by using `+` instead of `*`. Otherwise, this will also match `[]()`.